src/mpvis/mpdfg/diagrammers/graphviz.py

```diff
@@ -207,5 +198,35 @@ def build_activity_name(self, activity_name: str) -> str:
 
         return activity_name
 
+    def get_arc_thickness_for_connection(self, connection):
+        """Calculate arc thickness for regular connections based on arc_thickness_by parameter."""
+        if self.arc_thickness_by == "frequency":
+            if "frequency" in self.dfg["connections"][connection]:
+                return link_width(
+                    self.dfg["connections"][connection]["frequency"],
+                    self.connections_dimensions_min_and_max["frequency"],
+                )
+            return 1
+        elif self.arc_thickness_by == "time":
+            if "time" in self.dfg["connections"][connection]:
+                return link_width(
+                    self.dfg["connections"][connection]["time"],
+                    self.connections_dimensions_min_and_max["time"],
+                )
+            return 1
+        else:  # "none" or any other value
+            return 1
+    
+    def get_arc_thickness_for_extreme(self, frequency):
```

Comment:
[REQ]
Make function logic cleaner. Something like this can work:

```suggestion
    def get_arc_thickness_for_extreme(self, frequency):
        if self.arc_thickness_by == "frecuency" and isinstance(frequency, (int, float)):
            return link_width(frequency, self.connections_dimensions_min_and_max["frequency"])
        return 1 # for every other case return value will be 1
```
I think there is no need to check "frequency" key in self.connections_dimensions_min_and_max.